communications/chat.py

```diff
@@ -1,11 +1,44 @@
 from channels.generic.websocket import AsyncWebsocketConsumer
 import json
-
+from communications.mongo_models import Room, Message
+from datetime import datetime
+from django.utils.timezone import now
 
 class Chat(AsyncWebsocketConsumer):
     async def connect(self):
+        self.room_id = self.scope['url_route']['kwargs']['room_id']
+        self.room_group_name = f"chat_{self.room_id}"
+
+        #питання чи робити канал чату для групи інвесторів з стартапом чи залишити чат тільки між інвестором і стартапом?
+        await self.channel_layer.group_add(self.room_group_name, self.channel_name)
         await self.accept()
-        await self.send(text_data=json.dumps({"message": "Connected!"}))
+        await self.send(json.dumps({"message": "Connected!"}))
+
+    #незнаю чи робити видалення чату після кінця disconnect() , чи це зайве???
```

Comment:
No, you shouldn't delete the chat on disconnect.
_Why?_
- `disconnect()` is triggered when a user closes the browser or loses connection.
- You risk accidentally deleting chats other users are still participating in.
- Also, messages are already stored in MongoDB, so chats can persist.

_Instead:_
Use a cron job or admin feature to archive or delete inactive chats after X days.